Project.toml

```
@@ -14,11 +15,20 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 
+[compat]
+JSON = ">= 0.4.0"
```

We should block on minor versions since packages can otherwise be breaking:
```suggestion
JSON = "0.4"
```
etc

---

I'm not very familiar with the Distributions, though. Given `REQUIRE` is outdated, should I pin to the latest package version? E.g., `QuadGK = "2.0"`, regardless of patch version

---

yes, we should not commit to patch version (those should be upgraded to the latest possible)